hw/ip/aes/model/aes_model.core

```diff
@@ -0,0 +1,26 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:model:aes:0.5"
+description: "AES Model"
+filesets:
+  files_cc:
+    files:
+      - crypto.c
+      - crypto.h: { is_include_file: true }
+      - aes.c
+      - aes.h: { is_include_file: true }
+    file_type: cppSource
+
+targets:
+  default:
+    filesets:
+      - files_cc
+
+    tools:
+      verilator:
+        mode: cc
+        verilator_options:
+# linker flags
+          - '-LDFLAGS "-pthread -lcrypto"'
```

Comment:
these linker flags won't be used, but I'm fine with having them here as documentation.

Reply:
> @vogelpi
> If I add my suggested edits I can compile further.
> I now need
> 
> ![image](https://user-images.githubusercontent.com/53917183/68590922-ef4f2f80-048f-11ea-8276-9003bd5a0f29.png)
> which I think might be a local issue.

You might want to try compiling the C model standalone using the makefile. See `hw/ip/aes/model/README.md`.